src/java/htsjdk/samtools/SAMSequenceDictionary.java

```diff
@@ -163,8 +165,72 @@ public boolean equals(Object o) {
         return true;
     }
 
+    /** return a MD5 sum for ths dictionary, the checksum is not re-computed each time this
+     * method is called. <pre>md5( (seq1.name+seq1.length) + ' '+(seq2.name+seq2.length) + ' '+...)</pre>
+     * @return a MD5 checksum for this dictionary or the empty string if it is empty
+     */
+    public String md5() {
+        if(isEmpty()) return "";
```

Comment:
@lindenb the coding style and elsewhere needs to be consistent with htsjdk.  It makes it easier for folks to maintain and read the code.  In this case, ```if(``` -> ```if (```.  Thanks for updating the code in this pull requests and future ones too, as I really appreciate the contributions you are making.  I would recommend using intelliJ's or your favorite IDE's code re-format function.